samples/grids/data-grid/cell-editing/wwwroot/TemplateColumnScript.js

```diff
@@ -0,0 +1,25 @@
+﻿function onUpdatingTemplateColumn(grid, args) {
+    let content = args.content;
+     
```

Comment:
please rename this file to "DataGridCellEditing.js" because we name JS files based on name of samples. This way, we have do not have generic names of JS files and we can identify JS files in browser when there is an issue.

Reply:
see commit 2